src/Psysh/Caster.php

```diff
@@ -1,9 +1,15 @@
 <?php
-
+// phpcs:ignoreFile
```

Comment:
I feel your pain, but I'd rather not start exempting files from our PSR-12 coding standard. Instead lets disable sniffs that are especially bothersome.

Reply:
Here's what it found. Some of this I can fix if you would confirm you are doing significantly different coding standards to Drupal. But at the same time, some of this just comes from the peculiar needs of this routine.


```
FILE: /home/wodby/drush/src/Psysh/Caster.php
----------------------------------------------------------------------
FOUND 21 ERRORS AFFECTING 13 LINES
----------------------------------------------------------------------
  32 | ERROR | [x] Opening parenthesis of a multi-line function call
     |       |     must be the last content on the line
     |       |     (PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket)
  45 | ERROR | [x] TRUE, FALSE and NULL must be lowercase; expected
     |       |     "false" but found "FALSE"
     |       |     (Generic.PHP.LowerCaseConstant.Found)
  45 | ERROR | [x] TRUE, FALSE and NULL must be lowercase; expected
     |       |     "true" but found "TRUE"
     |       |     (Generic.PHP.LowerCaseConstant.Found)
 142 | ERROR | [x] Opening brace should be on a new line
     |       |     (Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine)
 147 | ERROR | [x] Opening parenthesis of a multi-line function call
     |       |     must be the last content on the line
     |       |     (PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket)
 148 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 12 spaces but found 16
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 148 | ERROR | [x] Only one argument is allowed per line in a
     |       |     multi-line function call
     |       |     (PSR2.Methods.FunctionCallSignature.MultipleArguments)
 148 | ERROR | [x] Only one argument is allowed per line in a
     |       |     multi-line function call
     |       |     (PSR2.Methods.FunctionCallSignature.MultipleArguments)
 148 | ERROR | [x] Closing parenthesis of a multi-line function call
     |       |     must be on a line by itself
     |       |     (PSR2.Methods.FunctionCallSignature.CloseBracketLine)
 149 | ERROR | [x] Expected 1 space after closing brace; newline
     |       |     found
     |       |     (Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace)
 151 | ERROR | [x] Opening parenthesis of a multi-line function call
     |       |     must be the last content on the line
     |       |     (PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket)
 152 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 12 spaces but found 16
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 152 | ERROR | [x] Only one argument is allowed per line in a
     |       |     multi-line function call
     |       |     (PSR2.Methods.FunctionCallSignature.MultipleArguments)
 152 | ERROR | [x] Closing parenthesis of a multi-line function call
     |       |     must be on a line by itself
     |       |     (PSR2.Methods.FunctionCallSignature.CloseBracketLine)
 153 | ERROR | [x] Expected 1 space after closing brace; newline
     |       |     found
     |       |     (Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace)
 155 | ERROR | [x] Opening parenthesis of a multi-line function call
     |       |     must be the last content on the line
     |       |     (PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket)
 156 | ERROR | [x] Multi-line function call not indented correctly;
     |       |     expected 12 spaces but found 16
     |       |     (PSR2.Methods.FunctionCallSignature.Indent)
 156 | ERROR | [x] Only one argument is allowed per line in a
     |       |     multi-line function call
     |       |     (PSR2.Methods.FunctionCallSignature.MultipleArguments)
 156 | ERROR | [x] Closing parenthesis of a multi-line function call
     |       |     must be on a line by itself
     |       |     (PSR2.Methods.FunctionCallSignature.CloseBracketLine)
 161 | ERROR | [x] Opening brace should be on a new line
     |       |     (Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine)
 169 | ERROR | [x] The closing brace for the class must go on the
     |       |     next line after the body
     |       |     (PSR2.Classes.ClassDeclaration.CloseBraceAfterBody

Reply:
Correct, we don't use Drupal standards. We use the coding standard that the rest of PHP community uses https://www.php-fig.org/psr/psr-12/.

Reply:
I have my opinion on the PHP community and their "standards" but those do not fit into polite conversation. Anyways, I will see what I can do.